stingray/lightcurve.py

```diff
@@ -177,13 +186,14 @@ def __init__(self, time, counts, err=None, input_counts=True,
 
         self.err_dist = err_dist
 
-        self.tstart = self.time[0] - 0.5*self.dt
-        self.tseg = self.time[-1] - self.time[0] + self.dt
+        self.tstart = np.min(self.time) - 0.5*self.dt
```

Comment:
My concern with these operations is that they are very expensive when the light curve is large. Assuming that the user has given the counts in temporal order should be safe, at least at __init__ time?